mmv1/products/databasemigrationservice/connectionprofile.yaml

```diff
@@ -336,15 +346,173 @@ properties:
           Output only. If the source is a Cloud SQL database, this field indicates the network architecture it's associated with.
         values:
           - :NETWORK_ARCHITECTURE_OLD_CSQL_PRODUCER
-          - :NETWORK_ARCHITECTURE_NEW_CSQL_PRODUCER -
-            !ruby/object:Api::Type::NestedObject
+          - :NETWORK_ARCHITECTURE_NEW_CSQL_PRODUCER
```

Comment:
The change was started with the post gres, but it was also definitely a typo that got through the original resource implementation so I don't see the harm in fixing it here rather than there so the change happens faster (especially since it isnt actually related to the functionality of that one)